diff --git a/autoload/SpaceVim/default.vim b/autoload/SpaceVim/default.vim
index f49d2800..277a79fe 100644
--- a/autoload/SpaceVim/default.vim
+++ b/autoload/SpaceVim/default.vim
@@ -44,7 +44,8 @@ function! SpaceVim#default#options() abort
   " nocompatible ,you need this
   set backspace=indent,eol,start
   set nrformats-=octal
-  set listchars=tab:→\ ,eol:↵,trail:·,extends:↷,precedes:↶
+  " set listchars=tab:→\ ,eol:↵,trail:·,extends:↷,precedes:↶
+  set listchars=tab:→\ ,eol:▸,trail:·,extends:▸,precedes:▸
   set fillchars=vert:│,fold:·
 
   set laststatus=2
diff --git a/config/plugins/tagbar.vim b/config/plugins/tagbar.vim
index e54924d6..815e358f 100644
--- a/config/plugins/tagbar.vim
+++ b/config/plugins/tagbar.vim
@@ -8,3 +8,4 @@ let g:tagbar_sort = get(g:, 'tagbar_sort', 0)
 let g:tagbar_compact = get(g:, 'tagbar_compact', 1)
 let g:tagbar_map_showproto = get(g:, 'tagbar_map_showproto', '')
 let g:tagbar_iconchars = ['▶', '▼']
+let g:tagbar_iconchars = ['▸', '▾']
